node/src/types/json_compatibility/contracts.rs

```diff
@@ -0,0 +1,118 @@
+// TODO - remove once schemars stops causing warning.
+#![allow(clippy::field_reassign_with_default)]
+
+use datasize::DataSize;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::types::json_compatibility::vectorize;
+use casper_types::{
+    Contract as DomainContract, ContractHash, ContractPackage as DomainContractPackage,
+    ContractPackageHash, ContractWasmHash, EntryPoint, NamedKey, URef,
+};
+use semver::Version;
```

Comment:
This should be with the other 3rd party imports.

Reply:
Regrouped